article.tex

```diff
@@ -810,6 +835,21 @@ \subsubsection*{Aromaticity}
 \end{verbatim}
 \vspace{0.2cm}
 
+For SMARTS matching and SMILES generation a model based
+on Daylight \cite{DaylightCIS} can be used and offers significant
+speed improvements to the one based on CDK Atom Types. 
+This model has recently been documented as part of the
+OpenSMILES specification (\url{http://opensmiles.org/}):
+
+\vspace{0.2cm}
+\begin{verbatim}
+Aromaticity aromaticity = new Aromaticity(
+  ElectronDonation.daylight(), Cycles.all()
+);
+\end{verbatim}
+\vspace{0.2cm}
+
```

Comment:
Sorry I still don't understand fully - no atom types here. The Daylight model is a superset of the CDK and doesn't require explicitly setting atom types. It doesn't use atom types (that's comp chem creeping in) but rather basic invariants of valence, charge, and bonding. All atoms are accepted by the model but only those in a narrow definition (in regards to the whole spectrum) are allowed to contribute. For the reason I would also recomend it. 👍 . It can actually be improved further which I guess would be... ``ElectronDonation.extended()`` or ``.cdk2()`` but not sure on naming.